client/src/services/fileOperationManager.ts

```diff
@@ -26,7 +26,7 @@ import {
 import { wait } from '@/parsec/internals';
 import { v4 as uuid4 } from 'uuid';
 
-const MAX_SIMULTANEOUS_IMPORT_JOBS = 5;
+export const MAX_SIMULTANEOUS_OPERATIONS = 3;
```

Comment:
This shouldn't be exported. We may change things a bit later on and allow some control over how many operations should be done simultaneously, but the components using the manager shouldn't rely on this variable, they should only rely on the events.